src/browser/api/application.js

```diff
@@ -438,14 +438,14 @@ Application.revokeWindowAccess = function() {
     console.warn('Deprecated');
 };
 
-Application.run = function(identity, configUrl = '') {
+Application.run = function(identity, userAppConfigArgs, configUrl = '') {
```

Comment:
Preferably yes. Had done it that way to avoid having to set a default but will just set the default to undefined.